Quick-Start [DA]/pg_cext.md

```diff
@@ -0,0 +1,148 @@
+# External Languages in PostgreSQL:
+External languages in PostgreSQL provide a powerful way to integrate additional programming languages, extending the database's capabilities beyond its native support. Developers can use these languages to implement custom functionality, access external libraries, and leverage their expertise in preferred programming languages.
+
+Using external languages offers several advantages:
+
+1. **Language Flexibility:** Integrate a wide range of languages like Python, Ruby, or Java to perform tasks that may not be as straightforward in SQL.
```

Comment:
I think Python, Ruby, and Java are all PLs, not external languages. Not sure what's really supported aside from C and maybe C++; whatever can generate a dynamic library, I would guess.

Reply:
yup your right, but just mentioning C and C++ doesn't really add value. 
Maybe could add languages like R or C#. Could you suggest some languages apart from this?